### DISCUSSION TITLE
Model Customisation?

### DISCUSSION THREAD

**Discussion by ghost** [open-webui/open-webui#3855] [General]
# Bug Report

## Description

**Bug Summary:**
System prompt in model customisation doesn't appear to work.

**Steps to Reproduce:**
Create a custom model based on an existing one. Enter a system prompt, such as "You are Ali G." (without the quotes). Select the model in a new chat.

**Expected Behavior:**
Expect the system prompt to cause the model to emit a relevant greeting, then emulate the role.

**Actual Behavior:**
No text emitted when new chat is started, and no system prompt customisation occurs.

## Environment

- **Open WebUI Version:** [e.g., 0.1.120]
- ? (Pulled 3-days ago on 11th July. Can't find version info anywhere convenient?)
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]
- ollama version is 0.2.3

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- Ubuntu 24.04
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]
- Chrome on distant Windows 11 machine.

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by justinh-rahb** [comment]
Update to Ollama 0.2.5, they had several issues with system prompts. It **should** be fixed now, but they already said so 4 other times 😆

**Comment by GrayXu** [comment]
@justinh-rahb 
Hi, I've encountered the same issue when using the OpenAI API. It seems that the system prompt only takes effect if set within each conversation setting (as observed from logs). Could this be an issue on the open-webui side?
